packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -296,21 +312,23 @@ export class VpcNetwork extends VpcNetworkRef {
             throw new Error('To use DNS Hostnames, DNS Support must be enabled, however, it was explicitly disabled.');
         }
 
+        this.tags = new cdk.TagManager(this, props.tags);
```

Comment:
I guess the `overwrite=false` is to allow consumers to set the `"name"` property? This seems like it's going to be a common case, so I wonder if it makes sense to add another constructor parameter to `TagManager` like `initialTags`, which would do this internally.

Reply:
I considered adding things like standard defaults (`Name = this.path`) was the one that came to mind. Then perhaps we just remove the `overwrite` property?

Reply:
Well, but you probably still want users to be able to pass in a custom `name` if they so desire, right? So the `overwrite` still needs to be there so we don't overwrite the consumer's customization.

I was just suggesting to make the pattern of "initial tags that can be overwritten" explit in the `TagManager` class.